snoop/models.py

```diff
@@ -40,7 +40,16 @@ class EmailCache(models.Model):
     value = models.TextField()
     time = models.DateTimeField(auto_now=True)
 
+class Collection(models.Model):
+    path = models.CharField(max_length=4000)
+    slug = models.CharField(max_length=100, db_index=True, unique=True)
+    title = models.CharField(max_length=200)
+    es_index = models.CharField(max_length=200)
+    description = models.TextField(blank=True)
+
 class Document(models.Model):
+    collection = models.ForeignKey('Collection',
+                                   related_name='document_set')
```

Comment:
`document_set` is the default `related_name`, isn't it? If so, let's not specify it.